src/components/Table/Table.jsx

```diff
@@ -395,7 +397,7 @@ const Table = props => {
       view.toolbar.customToolbarContent ||
       tooltip ? (
         <TableToolbar
-          tableId={id}
+          tableId={id || tableId}
```

Comment:
trying to understand what this is doing... basically if we are passed an ID then we use it, otherwise we use the tableId which comes from the state that is some randomly generated id?  Why is tableId something that is stateful?

Reply:
I think github auto collapsed the outdated conversation we had about it in from my review: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1470#discussion_r471747991

Reply:
@stuckless we are  using the useState hook to get a value that is not updated on re-renders. Since the default is only evaluated once and we do not provide an update method it will be unique for each table instance.